src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -28,95 +28,108 @@
+                                             (send-utils/reset-loading-network-amounts-to-zero
+                                              receiver-network-values)
+
+                                             (not= tx-type :tx/bridge)
+                                             send-utils/safe-add-type-edit))
+           network-links                 (when routes-available?
+                                           (send-utils/network-links chosen-route
+                                                                     sender-network-values
+                                                                     receiver-network-values))]
+       {:db (update-in db
+                       [:wallet :ui :send]
+                       assoc
+                       :suggested-routes          suggested-routes-data
+                       :route                     chosen-route
+                       :from-values-by-chain      from-network-amounts-by-chain
+                       :to-values-by-chain        to-network-amounts-by-chain
+                       :sender-network-values     sender-network-values
+                       :receiver-network-values   receiver-network-values
+                       :network-links             network-links
+                       :loading-suggested-routes? false
+                       :enough-assets?            true)}))))
+
+(rf/reg-event-fx
+ :wallet/suggested-routes-error
  (fn [{:keys [db]} [error-message]]
-   (let [cleaned-sender-network-values   (-> (get-in db [:wallet :ui :send :sender-network-values])
-                                             (send-utils/reset-loading-network-amounts-to-zero))
-         cleaned-receiver-network-values (-> (get-in db [:wallet :ui :send :receiver-network-values])
-                                             (send-utils/reset-loading-network-amounts-to-zero))]
-     {:db (-> db
-              (update-in [:wallet :ui :send] dissoc :route)
-              (assoc-in [:wallet :ui :send :sender-network-values] cleaned-sender-network-values)
-              (assoc-in [:wallet :ui :send :receiver-network-values] cleaned-receiver-network-values)
-              (assoc-in [:wallet :ui :send :loading-suggested-routes?] false)
-              (assoc-in [:wallet :ui :send :suggested-routes] {:best []}))
-      :fx [[:dispatch
-            [:toasts/upsert
-             {:id   :send-transaction-error
-              :type :negative
-              :text error-message}]]]})))
+   {:db (-> db
+            (update-in [:wallet :ui :send] dissoc :route)
+            (update-in [:wallet :ui :send :sender-network-values]
+                       send-utils/reset-loading-network-amounts-to-zero)
+            (update-in [:wallet :ui :send :receiver-network-values]
+                       send-utils/reset-loading-network-amounts-to-zero)
+            (assoc-in [:wallet :ui :send :loading-suggested-routes?] false)
+            (assoc-in [:wallet :ui :send :suggested-routes] {:best []}))
```

Comment:
This is a non-related refactor. I noticed we extracted the data, modified it, and the stored it in the same location. I refactored to use `update-in` and we get the same result with less code